RevitClasher/Tools.cs

```diff
@@ -65,27 +65,22 @@ public static List<ClashItems> clashingElements(Document doc, Application app, D
             Autodesk.Revit.DB.Options opt = new Options();
             var ActiveViewBB = doc.ActiveView as View3D;
 
-            BoundingBoxXYZ bbV = ActiveViewBB.GetSectionBox();
-
-
-            var VisibleLinkedElements = new List<GeometryElement>();
-
             //Hard coded selection
             LogicalOrFilter filterLinkedCategories = new LogicalOrFilter(FormTools.SelectedCategories);
             FilteredElementCollector collector = new FilteredElementCollector(SecondDocument).WherePasses(filterLinkedCategories).WhereElementIsNotElementType();
             linkedElements = collector.ToElements() as List<Element>;
 
             var ClashItemsList = new List<ClashItems>();
-            foreach (var item in linkedElements)
+            foreach (Element elementB in linkedElements)
```

Comment:
Renaming the variable to a more appropiate name. Element B indicates that this variable has data of the element that will receive clash from another element.